WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderCommentAdapter.java

```diff
@@ -133,13 +134,18 @@ public ReaderCommentAdapter(Context context, ReaderPost post) {
         int mediumMargin = context.getResources().getDimensionPixelSize(R.dimen.margin_medium);
         mContentWidth = displayWidth - (cardMargin * 2) - (contentPadding * 2) - (mediumMargin * 2);
 
-        mColorAuthor = context.getResources().getColor(R.color.blue_medium);
-        mColorNotAuthor = context.getResources().getColor(R.color.grey_dark);
-        mColorHighlight = context.getResources().getColor(R.color.grey_lighten_30);
+        mColorAuthor = getColorResource(context, R.color.blue_medium);
+        mColorNotAuthor = getColorResource(context, R.color.grey_dark);
+        mColorHighlight = getColorResource(context, R.color.grey_lighten_30);
 
         setHasStableIds(true);
     }
 
+    @SuppressWarnings("deprecation")
+    private int getColorResource(Context context, @ColorRes int colorResId) {
+        return context.getResources().getColor(colorResId);
+    }
+
```

Comment:
Should this be removed and the new ResourceUtils.getColorResource method be used in its place?